services/wiki/wiki.go

```diff
@@ -81,6 +86,36 @@ func InitWiki(repo *models.Repository) error {
 	return nil
 }
 
+// prepareWikiFileName try to find a suitable file path with file name by the given raw wiki name.
+// return: existence, prepared file path with name, error
+func prepareWikiFileName(gitRepo *git.Repository, wikiName string) (bool, string, error) {
```

Comment:
a smal unit test would be nice :D

Reply:
I'd like to but since I'm also not very familiar with the whole Gitea codebase and how these tests works, [I think it's better to not add a test case for this PR](https://github.com/go-gitea/gitea/pull/16139#issuecomment-860194111).

Reply:
Hmm ok. If i get time I'll write one, or at least a frame :)

Reply:
-> https://github.com/go-gitea/gitea/pull/16487